Raspi/Test/mqtt_wireless_log.py

```diff
@@ -70,17 +72,29 @@ def on_message(client, userdata, msg):
 
 def start_recording(module_id_str):
     """ Start recording for a specific module """
-    # Clear the old temporary files in the folder, just in case the script
-    # exited early and they were not cleared properly
-    remove_files(find_temp_csvs(module_id_str))
+    # The old temporary files in the folder will not be removed just incase
+    # they contain just in case they need to be recovered.
 
     # Save the state of recording and the output filename to global dicts
     is_recording[module_id_str] = True
     module_start_time[module_id_str] = datetime.now()
 
-    # Generate filename from the current date and time in the Australian format
-    output_filename[module_id_str] = module_start_time[module_id_str].strftime(
-        module_id_str + "_D%d-%m-%d_T%H-%M-%S" + ".csv")
+    # Generate filename from the last log number + 1
+    max_file_id = 0
+    for filepath in glob.glob('./../logs/*_M?.csv'):
+        # split the filepath into the filename
+        filename = filepath.split("/")[3]
+
+        # Gets all the digits from the file name
+        temp = re.findall(r'\d+', filename)
+
+        # Gets all the digits from the file name
+        file_id = list(map(int, temp))[0]
```

Comment:
As @hallgchris mentioned above, I think i'd be comfortable if this part of the code was unit tested somehow. (Definitely in your next PR since it's quite new to you)